src/development/ui/advanced/shaders.md

```diff
@@ -5,51 +5,65 @@ short-title: Fragment shaders
 ---
 
 {{site.alert.note}}
-  The HTML backend doesn't support the `FragmentProgram` API.
-  The `CanvasKit` backend will support this API in a future release.
+  The `CanvasKit` backend supports this feature on the `beta` branch
+  and should support this API in the next stable release.
+  There are no current plans to support this feature in the HTML backend.
 {{site.alert.end}}
 
-A shader is a program executed by the GPU and usually authored in a small,
-Dart-like language, such as GLSL. User-authored shaders can be added to
-Flutter projects using the [`FragmentProgram`][] API.
-You can use custom shaders to provide rich graphical effects beyond those
-provided by the Flutter SDK.
+Custom shaders can be used to provide rich graphical effects
+beyond those provided by the Flutter SDK.
+A shader is a program authored in a small, Dart-like language,
+known as GLSL,
+and executed on the user's GPU.
+
+User-authored shaders are added to a Flutter project
+by listing them in the `pubspec.yaml` file,
```

Comment:
Nit, here user refers to the end user, but the next line user refers to the developer.

Maybe specifitcally call out end-user for the first one?

Reply:
i'll replace it with "Custom" for consistency with the rest.

Normally we use "developers" for our users.